src/sentry/testutils/cases.py

```diff
@@ -377,6 +387,29 @@ def assertWriteQueries(self, queries, debug=False, *args, **kwargs):
         with context:
             func(*args, **kwargs)
 
+    def create_plugin(self, org=None):
+        org = org or self.organization
+        plugin, _ = PluginRegistration.objects.get_or_create(
+            name='tests_utils.create_plugin', version='1.0.0', organization=org)
+        return plugin
+
+    def register_extensible(self, klass, plugin=None):
+        plugin = plugin or self.create_plugin()
+        ret = self.app.extensibles.register(plugin, klass)
+        return ret
+
+    def create_substance(self, klass, name=None, **kwargs):
+        # TODO: This shouldn't be necessary as there should be only one app instance per test
```

Comment:
Isn't this fixed now?

Reply:
Yeah, don't really understand what I'm going on about there...